pkg/cmd/collect.go

```diff
@@ -41,6 +54,30 @@ var collectCmd = &cobra.Command{
 		}
 		utils.IfErrorExitOrPanic(err)
 
+		for _, c := range collectorNames {
+			if c == collectors.LogsCollectorName && logsOutputFile == "" {
+				utils.IfErrorExitOrPanic(utils.NewMissingInputError(
+					errors.New("if Logs collector is selected you must also provide a log output file")),
+				)
+			}
+		}
+
+		if strings.Contains(tempDir, "~") {
+			usr, err := user.Current()
+			if err != nil {
+				log.Fatal("Failed to result homedir for current user in --tempdir")
```

Comment:
Since the error is for `user.Current()` maybe we should rephrase the error message? something like
```suggestion
			usr, err := user.Current()
			if err != nil {
				log.Fatal("Failed to get current user")
```

Reply:
Changed to `"Failed to fetch current user so could not resolve tempdir"`